Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -9,6 +9,38 @@
 
 namespace Hazel {
 
+	template <size_t N>
+	struct CleanExpressionResult
```

Comment:
Rename to `ExpressionResult` or `FunctionSignatureResult`?

Reply:
Changed it to ChangeResult.